lib/Server.js

```diff
@@ -235,7 +235,10 @@ function Server(compiler, options) {
 		historyApiFallback: function() {
 			if(options.historyApiFallback) {
 				// Fall back to /index.html if nothing else matches.
-				app.use(historyApiFallback(typeof options.historyApiFallback === "object" ? options.historyApiFallback : null));
+				app.use(historyApiFallback(
+					typeof options.historyApiFallback === "object" ? options.historyApiFallback : null),
+					express.static(options.contentBase || process.cwd())
```

Comment:
`options.contentBase || process.cwd()` is also used in the `contentBase` method. Can you put this in a variable, so both use the same variable to get the content base?